src/extension/settings/settingsPropertyManager.ts

```diff
@@ -108,17 +108,28 @@ export class SettingsPropertyManager {
             profileObject = general.plainObjectToClass(ConanProfileConfiguration, selectedProfileObject);
         }
 
+        let pythonInterpreter = await python.getCurrentPythonInterpreter();
+        if (pythonInterpreter !== undefined) {
+            if (!profileObject?.conanPythonInterpreter) {
+                profileObject!.conanPythonInterpreter = pythonInterpreter;
+            }
+            if (!profileObject?.conanExecutable) {
+                profileObject!.conanExecutable = path.join(path.dirname(pythonInterpreter), "conan");
```

Comment:
This might work in case of virtual environment, in case of system wide python, the path where conan executable is located, is completely in different place.
I would suggest to check if the `conan` binary exist after building the path, otherwise set to empty string as before.